test/suite/test_backup10.py

```diff
@@ -115,7 +115,15 @@ def test_backup10(self):
             lambda:self.assertEquals(self.session.open_cursor(None,
             bkup_c, None), 0), msg)
 
-        bkup_c.close()
+        dupc = self.session.open_cursor(None, bkup_c, config)
+        dup_logs = self.take_log_backup(bkup_c, self.dir, orig_logs, dupc)
+        
+        msg = "/since duplicate cursor follows parent cursor/"
+        # Test closing backup cursor before duplicate backup cursor
+        self.assertRaisesWithMessage(wiredtiger.WiredTigerError,
+            lambda:self.assertNotEqual(bkup_c.close(), 0), msg)
+
+        dupc.close()
```

Comment:
If we go ahead with the change, then I don't think you need/want the assert in there. I think this can just be:
self.assertRaisesWithMessage(wiredtiger.WiredTigerError, lambda:bkup_c.close(), msg)